apollo-testing-support/api/apollo-testing-support.api

```diff
@@ -18,11 +18,17 @@ public final class com/apollographql/apollo3/testing/ReadFileJvmKt {
 	public static final fun shouldUpdateTestFixtures ()Z
 }
 
-public final class com/apollographql/apollo3/testing/RunWithMainLoopJvmKt {
-	public static final fun runTest (Lkotlin/coroutines/CoroutineContext;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;)V
-	public static synthetic fun runTest$default (Lkotlin/coroutines/CoroutineContext;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;ILjava/lang/Object;)V
+public final class com/apollographql/apollo3/testing/RunTestKt {
 }
 
 public final class com/apollographql/apollo3/testing/TestNetworkTransportKt {
 }
 
+public final class com/apollographql/apollo3/testing/internal/RunTestBlockingKt {
+	public static final fun runTestBlocking (Lkotlin/coroutines/CoroutineContext;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;)V
+	public static synthetic fun runTestBlocking$default (Lkotlin/coroutines/CoroutineContext;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;Lkotlin/jvm/functions/Function2;ILjava/lang/Object;)V
+}
```

Comment:
I believe this is a false positive from the binary compatibility tracker as they are marked `@ApolloInternal`